triggers/simulations/overview.rst

```diff
@@ -0,0 +1,461 @@
+  - external processes to interact with the ledger - e.g. to simulate external code (or ledger workloads) creating or archiving contracts.
+
+  Ledger processes make no attempt at retrying failed command submissions. This is a known limitation.
+
+Each trigger simulation needs to define a single ledger process as follows:
+
+.. code-block:: scala
+  override protected def triggerMultiProcessSimulation: Behavior[Unit] = {
+    implicit def applicationId: ApiTypes.ApplicationId = this.applicationId
+    
+    Behaviors.setup { context =>
+      val setup = for {
+        client <- defaultLedgerClient()
+        party <- allocateParty(client)
+      } yield (client, party)
+      val (client, actAs) = Await.result(setup, simulationConfig.simulationSetupTimeout)
+      // Ledger process (as an Akka typed actor) being defined
+      val ledger: ActorRef[LedgerProcess.Message] =
+        context.spawn(LedgerProcess.create(client), "ledger")
+
+      // Trigger and external components could be defined here
+
+      Behaviors.empty
+    }
+  }
+
+Simulating External Ledger Interactions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As external components may interact with a ledger (e.g. by creating or archiving contracts that a trigger registers an interest in), it is often necessary to model these within a given trigger simulation. This may be done by defining an Akka typed actor with type ``Behavior[Unit]`` and having this actor send ``LedgerProcess.ExternalAction`` messages to the ledger actor.
+
+For example, to model an external component that randomly creates instances of a ``Cat`` contract once every second, we could write:
+
+.. code-block:: scala
+  def createRandomCat(
+    ledger: ActorRef[LedgerProcess.Message],
+    actAs: Party
+  )(implicit materializer: Materializer): Behavior[Unit] =
+    Behaviors.withTimers[Unit] { timer =>
+      timer.startTimerWithFixedDelay((), 1.second, 1.second)
+
+      Behaviors.receiveMessage { _ =>
+        val randomCat =
+          CreatedEvent(
+            templateId = Some("Cats:Cat"),
+            createArguments = Some(SValue.SRecord(
+              "Cats:Cat",
+              fields = ImmArray("owner", "name"),
+              values = ArrayList(
+                SValue.SParty(actAs),
+                SValue.SInt64(Random.nextLong()),
+              ),
+            )),
+        )
+        val createEvent = LedgerProcess.ExternalAction(CreateContract(randomCat, actAs))
+
+        ledger ! createEvent
+        Behaviors.same
+      }
+    }
+
+where a ``Cat`` template might be defined as:
+
+.. code-block:: none
+  template Cat
+    with
+      owner : Party
+      name : Int
+    where
+      signatory owner
+
+.. note::
+  Currently, it is not possible to model external components that exercise choices on a contract. This is a known limitation.
+
+  Currently, a level of understanding of the low level Speedy machine ``SValue`` is required when defining create or archive events. This is a known limitation.
```

Comment:
First use of `level` is vague and clashes with the use of the word `level` later - we can just use `good` here
`Currently, a level of understanding of` -> `Currently, a good understanding of`